### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by acBerger**
I tried to write out the test cases.

### CODE REVIEW

**Comment by RomanLangrehr** [comment]
> I tried to write out the test cases.

I think there is something wrong with our PR. According to the diff, you only changed some refernces...

**Comment by acBerger** [comment]
yes, it is. I am trying to correct in

**Comment by acBerger** [comment]
[PDF] (https://github.com/acBerger/Beagle/blob/gh-pages/branches/testing/Requirements%20Specification.pdf)

File: doc/Requirements Specification/Parts/Test Cases.lyx [hunk 1]
```diff
@@ -143,7 +143,22 @@ name "/T20/"
...
+\begin_layout Plain Layout
+
+
+\backslash
+gls{PCM}
+\end_layout
+
+\end_inset
+
+ it is essential to have a correct measurement.
```
Comment (jGleitz) [inline doc/Requirements Specification/Parts/Test Cases.lyx]: maybe “it is essential to measure at the correct points” fits better?

**Comment by jGleitz** [comment]
lgtm

**Comment by viridispirus** [comment]
lgtm

**Pull request merged**